src/views/bookmark_x/controller.ts

```diff
@@ -544,9 +605,9 @@ export class Controller {
         let wsf = this.get_wsf_with_node(group);
         if (group.children.length > 0) {
             let val = await
-            window.showInformationMessage(
-                "the group not empty, do you really want to delete it?", 'yes', 'no'
-            );
+                window.showInformationMessage(
+                    "the group not empty, do you really want to delete it?", 'yes', 'no'
+                );
```

Comment:
what kind of formatter r u using, I'll set a formatter for the project later, but now, if it is not a serious formatting problem, you can ignore it for now

Except for the above formatting changes, all others are reasonable.

currently, I'm using 
![image](https://github.com/user-attachments/assets/a714090f-587a-4ab9-8584-b791371aa2b8)